### ISSUE TITLE
Qwen2_5_VLVisionAttention attention_mask is None

### ISSUE THREAD

**Issue by JamesCao2048** [huggingface/transformers#42271]
https://github.com/huggingface/transformers/blob/a5c903f877fda21e739027eed133e03162eb7712/src/transformers/models/qwen2_5_vl/modeling_qwen2_5_vl.py#L250C21-L250C39

The attention_mask is None. When I use SDPA attention, the sdpa_attention_forward in transformers/integrations/sdpa_attention.py also not set.attention mask.

**Comment by zucchini-nlp** [comment]
Qwen-VL vision module does not need attention mask because the inputs are not padded. Internally the model packs variable length inputs, and in attention blocks we iterate over each input when SDPA attention

Not a bug, closing as resolved
